app/eventyay/plugins/sendmail/templates/pretixplugins/sendmail/outbox_list.html

```diff
@@ -0,0 +1,164 @@
+{% extends "pretixcontrol/event/base.html" %}
+{% load i18n %}
+{% load sendmail_extras %}
+
+{% block content %}
+<nav id="event-nav" class="header-nav">
+    <div class="navigation">
+        <div class="navigation-title">
+            <h1>
+                {% blocktranslate count mail_count=mails|length %}
+                    Queued email
+                {% plural %}
+                    {{ mail_count }} Queued emails
+                {% endblocktranslate %}
+            </h1>
+        </div>
+        {% include "pretixcontrol/event/component_link.html" %}
+    </div>
+</nav>
+
+<div class="submit-group">
+    <div class="row">
+        <div class="col-md-12">
+            <form method="get" class="form-inline pull-left">
+                <div class="form-group">
+                    <input type="text" name="q" class="form-control"
+                           placeholder="{% translate 'Search' %}" value="{{ query }}">
+                </div>
+                {% if current_ordering %}
+                    <input type="hidden" name="ordering" value="{{ current_ordering }}">
+                {% endif %}
+                <div class="form-group">
+                    <button type="submit" class="btn btn-primary">
+                        {% translate "Search" %}
+                    </button>
+                </div>
+            </form>
+
+            <div class="btn-toolbar pull-right" role="toolbar">
+                <div class="btn-group" role="group">
+                </div>
+                <div class="btn-group" role="group">
+                    <a href="{% url 'plugins:sendmail:purge_all' organizer=request.event.organizer.slug event=request.event.slug %}" class="btn btn-cancel">
+                        {% translate "Discard all" %}
+                    </a>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="clearfix"></div>
+</div>
+
+<table class="table mt-4">
+    <thead>
+        <tr>
+            {% for field, label in headers %}
+                <th>
+                    {{ label }}
+                    {% with asc=field desc="-"|add:field %}
+                        <a href="?ordering={{ asc }}{% if query %}&q={{ query }}{% endif %}"
```

Comment:
It is a bad practice to compose URL query string in template. You should do it in Python code, processing it with [`urllencode`](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlencode), before passing down to template, because some characters are illegal in URL.